Rewrite JS3 week 1 prep (#351)
CodeYourFuture/curriculum: content/js3/blocks/single-datum/index.md

+++
title = '💽 Single datum'
headless = true
time = 30
facilitation = false
emoji= '🧩'
[objectives]
    1='Describe how to render a single item of data in the browser'
+++

> To break down this problem, let's define a sub-goal:

Reviewer: ```suggestion
```

> 🎯 Sub-goal: Render a single datum in the user interface

> We'll consider how to render a single datum before doing this for the whole list. We will start with an object like the one below:

Reviewer: ```suggestion
To break down this problem, we'll render a single datum, before doing this for the whole list. Here's one show:
```

+++
title = '💽 Single datum'
headless = true
time = 30
facilitation = false
emoji= '🧩'
[objectives]
    1='Define a sub-goal for rendering data in the user interface'
+++

> 🎯 Sub-goal: Build a film card component

To break down this problem, we'll render a single datum, before doing this for the whole list. Here's one film:

```js
const film = {
    title: "Killing of Flower Moon",

>     director: "Martin Scoresese"

Reviewer: ```suggestion
    director: "Martin Scorsese"
```
